src/main/java/nextstep/ladder/Line.java

```diff
@@ -0,0 +1,36 @@
+package nextstep.ladder;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Line {
+  private final List<Boolean> points;
+
+  public Line(int count) {
+    this.points = generatePoints(count);
+  }
+
+  private List<Boolean> generatePoints(int count) {
+    List<Boolean> result = new ArrayList<>();
+    Random random = new Random();
+
+    for (int i = 0; i < count - 1; i++) {
+      result.add(canDrawLine(result, i) ? random.nextBoolean() : false);
+    }
```

Comment:
```suggestion
    for (int i = 0; i < count - 1; i++) {
      result.add(canDrawLine(result, i) && random.nextBoolean());
    }
```
만약 직관적인 표현을 위해서라면, 별도 메서드로 한번 더 추출해서 이름을 부여해주는 것도 좋아보여요 😄